www/include/configuration/configObject/timeperiod/DB-Func.php

```diff
@@ -419,3 +419,63 @@ function getTimeperiodIdByName($name)
     }
     return $id;
 }
+
+/**
+ * Get chain of time periods via template relation
+ *
+ * @global \Pimple\Container $dependencyInjector
+ * @param array $tpIds
+ * @param array $result
+ * @return array
+ */
+function getTimeperiodsFromTemplate(array $tpIds, array $result = null)
+{
+    global $dependencyInjector;
+
+    $db = $dependencyInjector['centreon.db-manager'];
+
+    $result = [];
+
+    foreach ($tpIds as $tpId) {
+        $db
+            ->getRepository(Centreon\Domain\Repository\TimePeriodRepository::class)
+            ->getIncludeChainByParant($tpId, $result)
+        ;
```

Comment:
https://github.com/php-fig-rectified/fig-rectified-standards/blob/master/PSR-2-R-coding-style-guide-additions.md#method-chaining

Reply:
just put the semicolon on previous line :)

Reply:
done

Reply:
btw this code style is described in section Further considerations